matlab/schemas/+stimulus/Clip.m

```diff
@@ -0,0 +1,61 @@
+%{
+# Movie clip condition
+-> stimulus.Condition
+-----
+-> stimulus.MovieClip
+cut_after              : float           # (s) cuts off after this duration
+%}
+
+classdef Clip < dj.Manual & stimulus.core.Visual
+    
+    properties(Constant)
+        variation = '1'
+        movie_dir = '/Users/dimitri/stimuli'
```

Comment:
Yes, it should be `~/stimuli` but PsychToolbox does not understand `~`.  we should ask Manolis what it this should be.